Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -604,20 +604,21 @@ static void BPWritten(const BPCmd& bp)
       GeometryShaderManager::SetTexCoordChanged((bp.address - BPMEM_SU_SSIZE) >> 1);
     }
     return;
+
   // ------------------------
   // BPMEM_TX_SETMODE0 - (Texture lookup and filtering mode) LOD/BIAS Clamp, MaxAnsio, LODBIAS,
   // DiagLoad, Min Filter, Mag Filter, Wrap T, S
   // BPMEM_TX_SETMODE1 - (LOD Stuff) - Max LOD, Min LOD
   // ------------------------
   case BPMEM_TX_SETMODE0:  // (0x90 for linear)
-  case BPMEM_TX_SETMODE0_4:
-    TextureCacheBase::InvalidateAllBindPoints();
-    return;
-
   case BPMEM_TX_SETMODE1:
+    TextureCacheBase::SetBindPoint(bp.address & 0x3, false);
```

Comment:
Everyone will have their own opinion but I think it might be clearer if you left each individual case statement in with the appropriate call?  Ex:

```
case BPMEM_TX_SETMODE0:
  TextureCacheBase::SetBindPoint(0, false);
case BPMEM_TX_SETMODE1:
  TextureCacheBase::SetBindPoint(1, false);
case BPMEM_TX_SETMODE1_4:
   TextureCacheBase::SetBindPoint(5, false);
```